Add missing entries in README.md (#14351)
netdata/netdata: collectors/plugins.d/README.md

|[apps.plugin](/collectors/apps.plugin/README.md)|`C`|linux, freebsd|monitors the whole process tree on Linux and FreeBSD and breaks down system resource usage by **process**, **user** and **user group**.|
|[charts.d.plugin](/collectors/charts.d.plugin/README.md)|`BASH`|all|a **plugin orchestrator** for data collection modules written in `BASH` v4+.|
|[cups.plugin](/collectors/cups.plugin/README.md)|`C`|all|monitors **CUPS**|

> |[ebpf.plugin](/collectors/ebpf.plugin/README.md)|`C`|linux|monitors different metrics on environments using kernel internal functions.|

Reviewer: ```suggestion
|[ebpf.plugin](https:/.github.com/netdata/netdata/blob/master/collectors/ebpf.plugin/README.md)|`C`|linux|monitors different metrics on environments using kernel internal functions.|
```

@thiagoftsm  Thanks for the update, IMPORTANT: We are trying to push the guideline; use of GH absolute links in every markdown (https://github.com/netdata/netdata/pull/14344#issue-1561772522)

Author: I accepted all suggestions, thanks!

> |[go.d.plugin](https://github.com/netdata/go.d.plugin/)|`C`|all|monitors different services running on host.|

Reviewer 2: ```suggestion
|[go.d.plugin](https://github.com/netdata/go.d.plugin/)|`Go`|all|collects metrics from the system, applications, or third-party APIs.|
```

> |[go.d.plugin](https://github.com/netdata/go.d.plugin/)|`Go`|all|collects metrics from the system, applications, or third-party APIs.|

Reviewer: ```suggestion
|[go.d.plugin](https://github.com/netdata/go.d.plugin/blob/master/README.md)|`Go`|all|collects metrics from the system, applications, or third-party APIs.|
```

Same here
